wordpress-mobile/WordPress-Android
# Issue/16150 blogging prompt card menu and additional buttons

#16150 part 2 of 3

This PR adds "Answered" and "Share" buttons to the bottom of the blogging prompt card, and an overflow menu.

For testing purposes clicking on Answer buttons shows "answered" state buttons. Clicking on "Answered" shows the "Answer" button again. Share button is wired to share action with the content of the prompt as a message (I'm not sure at this point how it will work in the future, but for now the prompt message is just passed back from VH). Rest of the buttons do not do anything at the moment.

Some parts of the layout are still WIP, but I'm starting to clean it up.

[![Image from Gyazo](https://i.gyazo.com/185e480350154196338c8253e1ef96da.png)](https://gyazo.com/185e480350154196338c8253e1ef96da)

[![Image from Gyazo](https://i.gyazo.com/ff14e06e7ca1c9560d7d942c575e884a.png)](https://gyazo.com/ff14e06e7ca1c9560d7d942c575e884a)

To test:
- Using wasabi version build variant, navigate to App Settings -> Debug settings and enable `blogging_prompts_remote_field` option.
- Confirm that clicking on "Answer" button of BP card shows "Answered" and "Share" buttons like the are displayed in Figma.
- Confirm that tapping on Share buttons shows an intent picker with the content of a prompt as a message.
- Confirm that tapping on overflow menu shows a menu that looks like the one in Figma.


## Regression Notes
1. Potential unintended areas of impact
- Changes are contained to the card, so should be none.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Automated and manual tests.

3. What automated tests I added (or what prevented me from doing so)
Testing that clicking on Share button calls appropriate method.

PR submission checklist:
